build/dev.js

```diff
@@ -0,0 +1,44 @@
+var colors = require('colors');
```

Comment:
this is not used anywhere.

Reply:
It's a weird package, requiring it adds prototype methods to the `String` class, such as the `.black.bgCyan` on line 12.

Reply:
I think I can probably not assign the return value to a const though, I'll make a revision here.